lib/src/ultis/helper.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter_device_type/flutter_device_type.dart';
+
+class ScreenUtil {
+  double height(double designedPixel) {
+    return designedPixel /
+        (Device.get().isTablet ? 1024 : 812) *
+        Device.screenHeight;
```

Comment:
what happens if the user device screen not suitable for this config?

Reply:
Can you give me some examples which this config can be wrong?